cms/templates/js/content-group-editor.underscore

```diff
@@ -7,13 +7,40 @@
     <div class="wrapper-form">
         <fieldset class="collection-fields">
             <div class="input-wrap field text required add-collection-name <% if(error && error.attributes && error.attributes.name) { print('error'); } %>">
-                <label for="group-cohort-name-<%= uniqueId %>"><%= gettext("Content Group Name") %></label>
+                <label for="group-cohort-name-<%= uniqueId %>"><%= gettext("Content Group Name") %></label><%
+                if (!_.isUndefined(id) && !_.isEmpty(id)) {
+                    %><span class="group-configuration-id">
+                        <span class="group-configuration-label"><%= gettext('Content Group ID') %></span>
+                        <span class="group-configuration-value"><%= id %></span>
+                    </span><%
+                }
+                %>
                 <input name="group-cohort-name" id="group-cohort-name-<%= uniqueId %>" class="collection-name-input input-text" value="<%- name %>" type="text" placeholder="<%= gettext("This is the name of the group") %>">
             </div>
         </fieldset>
+        <% if (!_.isEmpty(usage)) { %>
+            <div class="wrapper-group-configuration-validation usage-validation">
+                <i class="icon fa fa-warning"></i>
+                <p class="group-configuration-validation-text">
+                    <%= gettext('This content group is currently used in different problems.') %>
```

Comment:
@explorerleslie @muzaffaryousaf when is this message displayed? I'm not clear what is meant by "different problems". Do we mean "more than one problem"? And should we be consistent in referring to content groups being in use in *units*, not *problems*?